test/quantization/core/test_quantized_op.py

```diff
@@ -255,18 +255,6 @@ def test_qrelu(self):
         ]
         devices = ["cpu", "cuda"] if TEST_CUDA else ["cpu"]
         for device in devices:
-            # Only test the non-in-place version relu quantized cuda,
```

Comment:
thought I left a review comment here but can't find it. I'm wondering why we add this in the previous PR and remove it here?

Reply:
For the previous PR, we only have non-in-palce qrelu, so I add this for only test non-in-palce qrelu, avoid in-palce qrelu.